grouper/expiration.py

```diff
@@ -49,7 +67,6 @@ def add_expiration(session, expiration, group_name, member_name, recipients, mem
         "member_name": member_name,
         "member_is_user": member_is_user,
     }
-    from grouper.fe.settings import settings as fe_settings
```

Comment:
Know of any reason this was inlined in the first place? Seems like it could be intentional

Reply:
I think it's because `grouper.fe.settings` and `grouper.settings` are separate singletons and, depending on the application, only one of them is refreshed in the background. But I have an upcoming diff that rips out the background refresh anyway, so it won't matter.